tests/test_base.py

```diff
@@ -0,0 +1,72 @@
+from flask_testing import TestCase
+import json
```

Comment:
`import os` should come first. Consider having your imports in the following order:

1. system imports.
2. package imports.
3. local file imports